icarus_backend/settings.py

```diff
@@ -91,7 +94,7 @@
     #     'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
     # }
     'default': {
-        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+        'ENGINE': 'django.contrib.gis.db.backends.postgis',
```

Comment:
are we sure that this will not fuckup anything in the future?

Reply:
Geodjango is not built by a 3rd party, it's a pre-bundled feature that's part of Django's contrib package. I have no proof it will not fuck up anything, I am just putting the same trust into Geodjango as I put into Django.